gravitee-apim-console-webui/src/management/api/policy-studio/debug/policy-studio-debug.component.spec.ts

```diff
@@ -0,0 +1,207 @@
+
+    httpTestingController.expectOne(`${CONSTANTS_TESTING.env.baseURL}/policies?expand=icon&withResource=false`).flush(policies);
+  });
+
+  afterEach(() => {
+    httpTestingController.verify();
+  });
+
+  it('should send simple debug request', fakeAsync(async () => {
+    const eventId = 'eventId';
+
+    const sendButton = await loader.getHarness(MatButtonHarness.with({ text: 'Send' }));
+
+    await sendButton.click();
+
+    tick();
+
+    expectSendDebugEvent(eventId);
+    tick(1000);
+    expectGetDebugEvent(eventId, false);
+    tick(1000);
+    expectGetDebugEvent(eventId, true);
+
+    fixture.detectChanges();
+
+    expect(fixture.nativeElement.querySelector('.policy-studio-debug-response__display-response__header').textContent).toContain(
+      'Response 200 - OK  GET /',
+    );
+  }));
+
+  describe('when debug response is success', () => {
+    const EVENT_ID = 'eventId';
+
+    beforeEach(fakeAsync(async () => {
+      const sendButton = await loader.getHarness(MatButtonHarness.with({ text: 'Send' }));
+      await sendButton.click();
+
+      tick();
+      expectSendDebugEvent(EVENT_ID);
+      tick(1000);
+      expectGetDebugEvent(EVENT_ID, true);
+
+      fixture.detectChanges();
+    }));
+
+    it('should display timeline', () => {
+      const timeLineCards = [...fixture.nativeElement.querySelectorAll('.policy-studio-debug-timeline-card').values()].map((card) => ({
+        content: card.querySelector('.policy-studio-debug-timeline-card__content').textContent,
+        state: card.querySelector('.policy-studio-debug-timeline-card__right').innerHTML.includes('gio:warning-circled-outline')
+          ? 'ERROR'
+          : card.querySelector('.policy-studio-debug-timeline-card__right').innerHTML.includes('gio:warning-circled-outline')
+          ? 'SKIPPED'
+          : undefined,
+      }));
+      expect(timeLineCards).toEqual([
+        {
+          content: ' Client APP ',
+        },
+        {
+          content: ' Request Input ',
+        },
+        {
+          content: ' Header  key-less ',
+        },
+        {
+          content: ' Header  policy-assign-attributes ',
+        },
+        {
+          content: ' Header  policy-override-request-method ',
+        },
+        {
+          content: ' Header  transform-headers ',
+        },
+        {
+          content: ' Header  transform-headers ',
+          state: 'ERROR',
+        },
+        {
+          content: ' Request Output ',
+        },
+        {
+          content: ' Backend Target ',
+        },
+        {
+          content: ' Response Input ',
+        },
+        {
+          content: ' Header  transform-headers ',
+        },
+        {
+          content: ' Body  policy-assign-content ',
+        },
+        {
+          content: ' Response Output ',
+        },
+        {
+          content: ' Client APP ',
+        },
+      ]);
+    });
+
+    it('should display inspector content for selected policy', () => {
```

Comment:
I added the nominal case "simple". 
but I think we should add the others 
- Error 
- Conditional 
- Input Request 
- Client App
or not I don't know 🤔